### PR TITLE
portfolio-performance: Add version 0.49.4

### PR DESCRIPTION
**Pull request by GotAnAccount**

### CODE REVIEW

**Comment by GotAnAccount** [comment]
any chance you could point to where I could find what fix is needed? is there a check list I missed?

File: bucket/portfolio-performance.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "shortcuts": [
+        [
+            "PortfolioPerformance.exe",
+            "Portfolio Performance"
+        ]
+    ]
```
Comment (Ash258) [inline bucket/portfolio-performance.json:25]:
```suggestion
    "version": "0.49.4",
    "description": "Investment portfolio performance calculator",
    "homepage": "https://www.portfolio-performance.info/en/",
    "license": "EPL-1.0 License",
    "suggest": {
        "JDK": [
            "java/oraclejdk",
            "java/openjdk"
        ]
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/buchen/portfolio/releases/download/0.49.4/PortfolioPerformance-0.49.4-win32.win32.x86_64.zip",
            "hash": "b6159cc003b6e6710715e4b0664945075803c8357a3e90f74db2c3a7dcec6ae3"
        }
    },
    "extract_dir": "portfolio",
    "shortcuts": [
        [
            "PortfolioPerformance.exe",
            "Portfolio Performance"
        ]
    ],
    "checkver": ">Download\\s+([\\d.]+)</",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/buchen/portfolio/releases/download/$version/PortfolioPerformance-$version-win32.win32.x86_64.zip"
            }
        }
    }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/portfolio-performance.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://www.portfolio-performance.info",
+    "description": "An open source tool to calculate the overall performance of an investment portfolio - across all accounts - using True-Time Weighted Return or Internal Rate of Return.",
+    "license": "EPL-1.0 License",
```
Comment (pratikpc) [inline bucket/portfolio-performance.json:4]: SPDX is EPL-1.0

File: bucket/portfolio-performance.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            "java/openjdk"
+        ]
+    },
+    "shortcuts": [
+        [
+            "PortfolioPerformance.exe",
+            "Portfolio Performance"
+        ]
+    ]
+}
```
Comment (pratikpc) [inline bucket/portfolio-performance.json:26]: We need checkver and autoupdate as well

checkver: "github" would work IMHO

**Pull request merged**